### ISSUE TITLE
Bug: NPE error preventing story arcs from progressing

### ISSUE THREAD

**Issue by AaronGullickson** [AaronGullickson/mekhq#20]
I have noticed that ocassionally, I am not having story arcs progress after the conclusion of a scenario and I get the following in the log:

```
java.lang.NullPointerException
	at java.base/java.util.Objects.requireNonNull(Objects.java:222)
	at mekhq.campaign.event.ScenarioEvent.<init>(ScenarioEvent.java:36)
	at mekhq.campaign.event.ScenarioChangedEvent.<init>(ScenarioChangedEvent.java:31)
	at mekhq.campaign.event.ScenarioResolvedEvent.<init>(ScenarioResolvedEvent.java:31)
	at mekhq.MekHQ.gameVictory(MekHQ.java:501)
	at megamek.common.event.GameVictoryEvent.fireEvent(GameVictoryEvent.java:65)
	at megamek.common.Game.processGameEvent(Game.java:2998)
	at megamek.client.Client.handlePacket(Client.java:1703)
	at megamek.client.Client$1.lambda$packetReceived$0(Client.java:176)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

Most recently, I noticed this with the Defend Ulric scenario on Wotan. However, it doesn't seem to happen consistently.

**Comment by AaronGullickson** [comment]
I ran through this in the debugger and that does appear to be the source of the problem. Moving the trigger immediately before the thread does not cause any problems, so I will do that.

**Comment by AaronGullickson** [comment]
Fixed in 3370fe29ee.
